libbeat/tests/system/idxmgmt.py

```diff
@@ -1,9 +1,10 @@
 from elasticsearch import NotFoundError
 from nose.tools import raises
 import datetime
+import unittest
 
 
-class IdxMgmt(object):
+class IdxMgmt(unittest.TestCase):
```

Comment:
I wonder why this didn't appear when we run the automatic conversion scripts :thinking:

Reply:
This wasn't strictly a python3 issue, I wanted to have a more descriptive assertion error by using `unittest.assertEqual` rather than `assert` which doesn't provide much in the way of understanding why the assertion failed.